core/modules/layout/includes/layout.layout.inc

```diff
@@ -25,26 +25,6 @@ function layout_layout_context_info() {
     // main context data.
     'load callback' => 'node_load',
   );
```

Comment:
What about moving this *node* layout context into `node.module` too ?

Reply:
I wondered about that. I'll give it a shot :)

Reply:
done!